src/main/java/com/google/step/servlets/JsonFormatter.java

```diff
@@ -13,6 +13,24 @@
 
 /** A class that handles converting entities to json format. */
 public class JsonFormatter {
+  public static String getCommentsWithTokenJson(
+      List<Comment> comments, List<User> users, String token) {
+    List<Map<String, Object>> commentMapList = new ArrayList<>();
+    for (int i = 0; i < comments.size(); i++) {
+      Comment comment = comments.get(i);
+      User user = users.get(i);
+      commentMapList.add(getCommentMap(comment, user));
+    }
```

Comment:
Maybe you can refactor this part to a function because they looks the same as line 36 - 41

Reply:
i just removed the getCommentsJson method, since it's not used anywhere